lib/SPIRV/libSPIRV/SPIRVDebug.h

```diff
@@ -72,7 +84,43 @@ inline spv_ostream &spvdbgs() {
 
 #define SPIRVDBG(x)
 
+// Minimal std::basic_ostream mock that ignores everything being printed via
+// operator<<
+class dev_null_stream {
+public:
+  void flush() {}
+};
+
+template <typename T>
+const dev_null_stream &operator<<(const dev_null_stream &Out, const T &) {
+  return Out;
+}
+
+template <typename T>
+const dev_null_stream &&operator<<(const dev_null_stream &&Out, const T &) {
+  return Out;
+}
+
+// Output stream for SPIRV debug information.
+inline dev_null_stream &spvdbgs() {
+  static dev_null_stream Out;
+  return Out;
+}
+
 #endif
 
+#define VERIFY_REGULARIZATION_PASS(Module, PassName)                           \
```

Comment:
Any reason why this has to be a macro?

Reply:
> Any reason why this has to be a macro?

No particular reason: I agree that macro is completely unnecessary, will refactor into a helper function instead

Reply:
Replaced macro with a helper function in 33c039b949569438afeba506ef8dfa7c56ed2a17